src/cloudant/client.py

```diff
@@ -73,7 +73,6 @@ def connect(self):
         if self._client_user_header is not None:
             self.r_session.headers.update(self._client_user_header)
         if not self.admin_party:
-            self.r_session.auth = (self._user, self._auth_token)
             self.session_login(self._user, self._auth_token)
```

Comment:
I would remove this `if` block altogether and replace it with just `self.session_login(self._user, self._auth_token)`.  There is a check in the session_login method which will handle the admin_party condition so there is no need to double up the "if" logic now.